webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3107,6 +3107,22 @@ var userOverride = {
                             'aria-label',
                             preLeadCollection.field
                         );
+                        kmChatInput.setAttribute(
+                            'pattern',
+                            "^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,7}$"
+                        );
+                        kmChatInput.setAttribute(
+                            'title',
+                            ''
+                        );
+                        kmChatInput.setAttribute(
+                            'oninvalid',
+                            `setCustomValidity("${MCK_LABELS['lead.collection'].errorEmail}")`
```

Comment:
can you please test this in IE?
It will break in IE as string template is not supported

Reply:
Yes I tested string template is not supported in IE, have changed the code accordingly.